card_kind.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Kind(Enum):
```

Comment:
np: I think the name `CardType` might be more suitable

Reply:
fixed